src/components/collection-progress-group/helpers/parse-collection-progress.ts

```diff
@@ -33,18 +32,6 @@ function parseCollectionProgress(
 	)
 	const isCompleted = completedTutorialCount == tutorialCount
 	const isInProgress = inProgressTutorialCount > 0 || completedTutorialCount > 0
-	/**
-	 * Tutorial CTA
-	 */
-	const tutorialCta = getNextTutorialCta({
```

Comment:
Refactored slightly to decouple `getNextTutorialCta` from `parseCollectionProgress`, as the former is only needed for collection-meta, and requires the full `tutorials` list, which we don't have access to in `collection-card`. Now `parseCollectionProgress` only needs `tutorialCount`, rather than the full array of `TutorialLite[]`.